src/components/PlioQuestion.vue

```diff
@@ -247,14 +265,16 @@ export default {
         this.checkAnswer();
         this.showResult();
       }, loadTime);
+
+      this.$emit("answer-submitted")
     },
 
     // Things to do after answer is submitted
     // and close button appears
     clickClose() {
       this.closeModal();
       this.isAnswerSubmitted = false;
-      this.$emit("answer-submitted", this.plioQuestion, this.selectedOption);
+      this.$emit("question-closed", this.plioQuestion, this.selectedOption);
```

Comment:
As explained in the comment above, now we know when submit was clicked and close was clicked